providers/src/airflow/providers/fab/www/extensions/init_appbuilder.py

```diff
@@ -500,7 +512,11 @@ def add_view_no_menu(self, baseview, endpoint=None, static_folder=None):
 
     @property
     def get_url_for_index(self):
-        # TODO: Return the fast api application homepage
+        if not self.enable_plugins and g.user is not None and g.user.is_authenticated:
+            # If plugins are disabled and the user is authenticated, it should be redirected to the Airflow 3 UI index page along with the JWT token
+            token = get_auth_manager().get_jwt_token(g.user)
+            return f"https://localhost:29091/webapp?token={token}"
```

Comment:
This is obviously not ideal. Should we create a config to store that value? Important point: it MUST be an HTTPS endpoint, otherwise FLask wont make the redirect

Reply:
Yes I think a config would be nice.

Reply:
I found a solution without creating a config